src/modules/configurator/components/ConfigPropertyDropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { ConfigProperty } from "./ConfigProperty";
+import { currentConfigurationsState, activePropState } from "../state";
+import { useRecoilValue } from "recoil";
+
+interface DropdownProps {
+  propertyName: "color" | "wheels";
+  isActive: boolean;
+}
+
+export function ConfigPropertyDropdown({
+  propertyName,
+  isActive,
+}: DropdownProps) {
+  const configurations = useRecoilValue(currentConfigurationsState);
+  const activePropIndex = useRecoilValue(activePropState);
+
+  const currentProperties =
+    propertyName === "color"
+      ? configurations?.color
+      : configurations?.wheelVariants;
+
+  return (
+    <ul>
+      {currentProperties &&
+        currentProperties.length > 0 &&
```

Comment:
`currentProperties?.length > 0`

Reply:
Changed to `Boolean(currentProperties?.length)`.